llvm/test/CodeGen/AMDGPU/stack-slot-color-after-sgpr-regalloc.mir

```diff
@@ -0,0 +1,353 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Should precommit the test. But I'm not sure this test currently does anything significant to the slot optimization.
Add another run line to show the output after stack-slot-coloring as well to see the effect of this pass.
Should add a short description at each test what it meant for?
I don't see the slot-indices being shared for the SGPR spills.

Reply:
I believe I added the RUN command for both BEFORE & AFTER stack slot coloring pass (the one invoked just after sgpr regAlloc), & stack indices re-usage can be seen.